base_report_assembler/__openerp__.py

```diff
@@ -34,11 +35,13 @@
 To install this assemblage option for specific object you can install
 the folling module(s):
 
-- Invoices: invoice_report_assemble (lp:account-invoice-report)
+- Assemble invoice report: `invoice_report_assemble`
+(git@github.com:OCA/account-invoice-report)
 
 """,
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
  'website': 'http://www.camptocamp.com/',
```

Comment:
@yvaucher  The website name should it be 'https://github.com/OCA/reporting-engine/'?